charts/be-web/values.yaml

```diff
@@ -278,3 +278,21 @@ monitoring:
     #     sum without(instance, pod) ( increase(whatever[5m]) ) > 0
     #   labels:
     #     severity: critical
+
+scaledobject:
+  spec: {}
+    # scaleTargetRef:
+    #   kind: Deployment
+    #   name: image-anonymizer
+    # minReplicaCount: 0
+    # maxReplicaCount: 1
+    # advanced:
+    #   restoreToOriginalReplicaCount: true
+    # triggers:
+    # - type: rabbitmq
+    #   metadata:
+    #     hostFromEnv: RABBITMQ_HOST # Optional. You can use this instead of `host` parameter
+    #     mode: QueueLength
+    #     value: "1"
+    #     queueName: sellers_media_to_anonymize
+    #     vhostName: sellers
```

Comment:
Don't you think it's better to provide an interface like
```
deployment:
    name: image-anonymizer

replicas:
    min: 0
    max: 1

triggers:
    rabbitmq:
        host: amqp://localhost:5672/sellers
        queueLength: 1
```

Reply:
maybe it's better if I change the comment and I just say to use the needed `spec`, with a link to the original documentation, WDYT?

Reply:
Yes, having it inside the `autoscaling` section make sense, I didn't thought about it.
But I would not abstract every line of the configuration, at this stage it seems to me as a big overengineering without advantages. The main spec part counts ~20 options ([doc](https://keda.sh/docs/2.10/concepts/scaling-deployments/)), plus the trigger part that depends on the single trigger (the [rabbitmq one](https://keda.sh/docs/2.10/scalers/rabbitmq-queue/) has another ~10 options)

Reply:
I mean that at this stage, I'd not limit nor direct the configuration of keda, as I have no experience with it, and the use case I have is very semplicistic. This means that I have no idea of what I'll need in the future, and what others will need in the future.
The options are:
- create an abstraction that serve just what I need (but for every other need we will need to change the abstraction, uncomfortable and time consuming, with no advantages)
- create an abstraction that replicates the full set of configuration (complex to develop now, complex to use as the dev has no direct correlation between the official documentation and our implementation, complex to maintain due to the double layer of abstraction. And no advantages)
- just put the configuration inside the values file. (easy to develop, easy to use, no maintenance). Of course inside the section you prefer

Reply:
@edubacco IMHO we can avoid abstracting the `triggers` section, but everything else can be inferred by the chart context (eg: you know if the target ref is a deployment or something else). Also, you can probably abstract part of the `TriggerAuthentication` object definition, as we already manage secrets/secret mounting. WDYT?

Reply:
Hmm I do not agree, I still think that creating the abstraction is complex to develop, and complex to use.
I.e.: the chart allows you to create a deployment and an indefinite number of worker pods.
Ho should i decide which is the target? If I just have a single deployment, probably it's not so complex, but what if I have one or two worker pods? I should ask the developer to specify in some way the target, and to let him specify it, why should I invent something different from the original configuration? I'll end up replicating it, or creating something more complex (and less usable/understandable, because there is a double abstraction layer)

Reply:
@edubacco ok, to avoid infinite discussions on this, let's do this: I gonna open up a PR over this one with an abstraction proposal so we can discuss on something real rather than keeping up with theoretical points.

Is that ok for you?

Reply:
Ok, I like the approach.
Can we also merge this, in the meantime? So we can stop blocking our team work. If we agree on another implementation, I'll change it also in the app using it

Reply:
If compatible with team's work, I would like to discuss #67 before merging to master and releasing this